java/main/src/test/java/com/saucelabs/saucebindings/integration/PerformanceTest.java

```diff
@@ -0,0 +1,168 @@
+package com.saucelabs.saucebindings.integration;
+
+import com.saucelabs.saucebindings.SauceSession;
+import com.saucelabs.saucebindings.options.SauceOptions;
+import com.saucelabs.saucebindings.performance.JankinessResults;
+import com.saucelabs.saucebindings.performance.PerformanceMetrics;
+import com.saucelabs.saucebindings.performance.PerformanceResults;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.openqa.selenium.By;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class PerformanceTest {
+    private SauceSession session;
+    private RemoteWebDriver driver;
+
+    @After
+    public void cleanUp() {
+        if (session != null) {
+            session.stop(true);
+        }
+    }
+
+    @Rule
+    public TestName testName = new TestName();
+
+    @Before
+    public void setup() {
+        SauceOptions sauceOptions = SauceOptions.chrome()
```

Comment:
Is `setCapturePerformance()` exposed on other browsers besides `chrome()`?

Reply:
Nope. Chrome only.